docs/rules/no-dupe-args.md

```diff
@@ -0,0 +1,22 @@
+# No duplicate arguments (no-dupe-args)
+
+If you have a function that takes many arguments you may sometimes use the same argument name more than once.
```

Comment:
Useful to mention that duplicate arguments are flagged in strict mode right up top here.